config/environments/development.rb

```diff
@@ -45,13 +52,16 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
-  # Allow requests for all domains e.g. <app>.dev.gov.uk
-  config.hosts.clear
+  # Raises error for missing translations.
+  # config.i18n.raise_on_missing_translations = true
```

Comment:
Should this be uncommented, as we previously had `config.action_view.raise_on_missing_translations` set to true?

Reply:
Thanks for spotting! Yep that should have been uncommented